### PR TITLE
No documentation about openshift-controller-manager-operator

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1724117

### CODE REVIEW

File: modules/nodes-cluster-controller-manager-operator-about.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  generations:
+    - group: apps
+      hash: ''
+      lastGeneration: 7
+      name: controller-manager
+      namespace: openshift-controller-manager
+      resource: daemonsets
+  observedGeneration: 3
+  readyReplicas: 0
+  version: 4.1.0-0.nightly-2019-07-31-005945
```
Comment (wzheng1) [inline modules/nodes-cluster-controller-manager-operator-about.adoc]: It's better not using nightly build in official document.

**Comment by wzheng1** [comment]
@adambkaplan @coreydaley  Could you please review and estimate what else introduction on openshift-controller-manager-operator is needed?

**Comment by adambkaplan** [comment]
@bmcelvee I'm inclined to NACK this. `openshift-controller-manager` is a control plane component, equivalent in importance to the OpenShift apiserver. Touching the operator configuration will very quickly lead customers to an unsupported state.

We do need docs on how to turn up logging for control plane components - customers used to do this by altering `master-config.yaml` and restarting the control plane components on each node.

**Comment by adambkaplan** [comment]
@wzheng1 @mburke5678 I think some of this content is appropriate for our README on Github (which currently has nothing): https://github.com/openshift/cluster-openshift-controller-manager-operator

**Comment by mburke5678** [comment]
Closing per @adambkaplan. Please let me know if we need to revisit this issue

**Pull request closed without merging**